index.html

```diff
@@ -87,112 +133,114 @@ <h3>LET'S CONNECT</h3>
         />
         <div class="project-section">
           <h3 class="project-name">Tonic</h3>
-          <div class="project-info">
+          <nav class="project-info">
```

Comment:
- [ ] adjust the spaces between your cards and right margin of your work section like the previous reviewer noted
![Screenshot (343)_LI](https://user-images.githubusercontent.com/65298048/186985195-6977dbcb-d3c2-4bf4-a52c-c525d80c9cae.jpg)